app/src/main/java/com/rmkrings/data/staff/StaffMember.java

```diff
@@ -15,9 +15,16 @@
 public class StaffMember {
     private final String name;
     private final ArrayList<String> subjects;
+    private final String email;
 
     StaffMember(JSONObject fromJSON) throws JSONException {
         name = fromJSON.getString("name");
+        boolean isTeacher = fromJSON.optBoolean("isTeacher");
```

Comment:
Since the boolean is only used 1 time, it does not need to be stored in a variable.